backend/lib/endpoints/schema/geo.js

```diff
@@ -21,8 +21,15 @@ const getLocationReverseGeocodeSchema = {
     .prop("lng", S.number().minimum(-180).maximum(180).required()),
 };
 
+const getHealthFacilityPlacesSchema = {
+  querystring: strictSchema()
+    .prop("lat", S.number().minimum(0).maximum(90).required())
```

Comment:
I am not sure but should this be minimum(-90)?

Reply:
Copied validity requirements from getLocationReverseGeocodeSchema.   Also  googled to verify and got this:  Latitude : max/min +90 to -90
Longitude : max/min +180 to -180.
But google maps throws error when lat is less than zero (I tested this) so I think that getLocationReverseGeocodeSchema requirements are probably right.   Josh M - Can you verify?  I can do further research if necessary.

Reply:
@joshmorel thoughts here?